shoes-package/spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 require 'pathname'
 require 'rspec/its'
 
-Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each {|f| require f}
+Dir["#{SHOES_PACKAGE_SPEC_ROOT}/support/**/*.rb"].each { |f| require f }
```

Comment:
also not too certain about the spaces here... but maybe it is indeed better and I should adapt :)